tag_manager/data_layer_name.rst

```diff
@@ -0,0 +1,59 @@
+.. highlight:: js
+.. default-domain:: js
+
+Custom data layer name
+======================
+
+Introduction
+------------
+The data layer is a global JavaScript object, that can be used to pass information form the website to PPMS container. The default value for the data layer name is `dataLayer`.
+
+    .. versionadded:: 10.0
+        The name of the object is customizable. The purpose of this article is to describe the steps that need to be performed to set up custom data layer name.
+
+Renaming the data layer
+-----------------------
+If you wish to rename your data layer, you should follow the instructions below:
+
+1. Log into your PPMS instance.
+#. Head to `Menu` > `Administration`.
+#. Select the website that you want to set the new data layer name for and then go to the `Installation` page.
+#. From here you should copy the code of the snippet, that you want to use - asynchronous or synchronous - and make the following change:
+
+    a. for the asynchronous snippet, you need to change the `dataLayer` value highlighted on a screen below
+
+        .. image:: ../_static/images/async-container.png
+            :alt: Container code for asynchronous tags
+
+    #. for the synchronous snippet, you need to input your desired name into the highlighted variable
+
+        .. image:: ../_static/images/sync-container.png
+            :alt: Container code for synchronous tags
+
+#. At this point, you can embed the snippet on your website. If you are replacing an existing snippet, make sure to take a look at the `Additional snippet migration steps`_ section for instructions on how to ensure full compatibility.
+
+.. note::
+    - The snippet code was slightly changed compared to PPMS versions pre 10.0, to accommodate for the possibility of changing the data layer name. The new snippet, released alongside version 10.0 is backward compatible, however, to use the functionality described in this document, whole snippet code needs to be replaced on your website.
+    - If you are using both synchronous and asynchronous snippets, it is recommended to use the same data layer name in both. Nonadherence to this rule may cause unexpected and unwanted behavior.
+    - We strongly advise making sure that the chosen name is not used in any other external software present on your website as well as inside your own website's code. Data layer name should be unique and reserved only for use inside the PPMS container. For additional guidelines regarding the data layer naming process, please refer to `Data layer name guidelines`_.
+
+
+Additional snippet migration steps
```

Comment:
I would name it as `Snippet migration` or `DataLayer name migration`. It should be clearer when to use it, so user can skip reading description if it doesn't affect him.